posthog/api/capture.py

```diff
@@ -46,6 +48,36 @@ def _load_data(request) -> Optional[Union[Dict, List]]:
     return data
 
 
+def _datetime_from_seconds_or_millis(timestamp: str) -> datetime:
+    if len(timestamp) > 11:  # assuming milliseconds / update "11" to "12" if year > 5138 (set a reminder!)
+        timestamp_number = float(timestamp) / 1000
+    else:
+        timestamp_number = int(timestamp)
+
+    return datetime.utcfromtimestamp(timestamp_number)
+
+
+def _get_sent_at(data, request) -> Optional[datetime]:
+    if request.GET.get('_'):  # posthog-js
```

Comment:
Do all of these variations exist in our libraries? If yes, maybe it is worth just adding comments which is which so we know

Reply:
I guess I was a bit too generous here. I checked the implementations and the only "sent_at" that are sent are `"sent_at"` for both iOS and Android and the `"_"` GET param for js.